packages/lib/modules/lbp/steps/useCreateLbpStep.tsx

```diff
@@ -91,30 +91,33 @@ export function useCreateLbpStep(): TransactionStep {
   const { symbol: launchTokenSymbol } = useTokenMetadata(launchTokenAddress, selectedChain)
   const { symbol: reserveTokenSymbol } = useTokenMetadata(reserveTokenAddress, selectedChain)
 
-  const createPoolInput = {
-    protocolVersion: 3 as const,
-    poolType: PoolType.LiquidityBootstrapping,
-    symbol: `${launchTokenSymbol}-${reserveTokenSymbol}-LBP`,
-    name: `${name} Liquidity Bootstrapping Pool`,
-    swapFeePercentage: parseUnits((fee / 100).toString(), 18),
-    chainId,
-    lbpParams: {
-      owner: owner || userAddress,
-      blockProjectTokenSwapsIn,
-      projectToken: launchTokenAddress as Address,
-      reserveToken: reserveTokenAddress as Address,
-      projectTokenStartWeight: parseUnits(`${projectTokenStartWeight / 100}`, 18),
-      reserveTokenStartWeight: parseUnits(`${reserveTokenStartWeight / 100}`, 18),
-      projectTokenEndWeight: parseUnits(`${projectTokenEndWeight / 100}`, 18),
-      reserveTokenEndWeight: parseUnits(`${reserveTokenEndWeight / 100}`, 18),
-      startTime: BigInt(Math.floor(new Date(startTime).getTime() / 1000)),
-      endTime: BigInt(Math.floor(new Date(endTime).getTime() / 1000)),
-    },
-  }
+  const createPoolInput =
+    startTime && endTime
```

Comment:
do we still have problems with the times not existing? don't we load this modal only when everything has been filled?

Reply:
yeah was still crashing w/o the time check and wanted to focus on safe wallet stuff so went back to the lazy fix

could spend more time figuring it out in another PR 

![image](https://github.com/user-attachments/assets/71c41ac0-2a28-4aa3-a3bc-e5670dabe488)